force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +87,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = '';
+
+        if (a == null) {
+            return null;
+        } else if (a > 0) {
+            result = 'Positive';
+        } else if (a < 0) {
+            result = 'Negative';
+        } else {
+            result = 'Zero';
+        }
+
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Good work. Consider the replace conditional with guard pattern I shared in Slack